### ISSUE TITLE
Different styles for different switch types

### ISSUE THREAD

**Issue by Wuzzy2** [Neverball/neverball#120]
Here's a small suggestion for Neverball:

I noticed there are different switch types: One-shot, toggle and timer.

The one-shot switch can only be activated once, if you roll again over it it won't be disabled. It is used in level 1 of Neverball Medium.
The toggle switch can be turned on and off as often as you like.
The timer is not important in this issue.

Both one-shot and timer types look identical, it would be nice if there is a small graphical difference, but this is not really important.

**Comment by fwp** [comment]
I agree that some visual distinction would be helpful. I think this issue has been discussed in the past but cannot quite remember why nothing became of it.

My personal preference would be to keep the star texture for one-time switches and use a different one (perhaps some kind of gear wheel texture) for those that can be toggled as in Hard 08 or 12.

Also note that, for historical reasons, there are still a number of official levels that use togglable switches where one-time switches might be better suited.
